org-roam.el

```diff
@@ -659,7 +725,36 @@ If item at point is not org-roam specific, default to Org behaviour."
                                                'file-from-point (plist-get properties :point))))
                                  (insert (format "%s \n\n" content)))))
                            backlinks))
-              (insert "\n\n* No backlinks!")))
+              (insert "\n\n* No backlinks!"))
+            (async-start
+             `(lambda ()
+                (setq load-path ',load-path)
+                (package-initialize)
+                (require 'org-roam)
+                (org-roam--find-unlinked-references ',file-path ',buffer-title ',(org-roam--find-all-files)))
+             (lambda (unlinked-references)
+               (let ((inhibit-read-only t))
+                 (with-current-buffer org-roam-buffer
+                   (if (> (length unlinked-references) 0)
+                       (progn
+                         (insert (format "\n\n* %d Unlinked references\n"
```

Comment:
Rather than inserting, maybe do a replace on everything under the "unlinked references" headline. Have a text property on the headline so you can check if the async return is for the correct file. 

```
(when (is-correct-file-p)
  (replace all content under "unlinked references" headline))
```